build/lib/locFunc.ts

```diff
@@ -0,0 +1,203 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Was this taken from somewhere?

Reply:
This is my own file that I created, using a combination of modified functions from i18n and extension.ts